### PR TITLE
Major changes in pl.js

### PR DESCRIPTION
**Pull request by ectryliz**

### CODE REVIEW

**Comment by ectryliz** [comment]
Missed some stuff, going to update them tomorrow.

**Comment by xhg4** [comment]
First of all, thanks for the massive contribution!
I will review the changes on Sunday (or at least that's the plan)
From what I've seen so far tho, the changes are generally good ones but I'll have to carefully look through them

**Comment by ectryliz** [comment]
For your information: I have fixed the consistency between translations, fixed all of the typos and punctuation errors. Changed what seem to be literal or incomprehensible translations and some small stylistic changes.

File: pl.js [hunk 1]
```diff
@@ -812,7 +812,7 @@ social.market.fee.gift Brak podatku
 social.market.gift Wyślij KR
 social.market.gift.msg Zostaw wiadomość
 respawn.auto Automatyczne odrodzenie za {0}
-custom.host.ranked Nie możesz hostować gier, gdy grasz w trybie rankowanym.
+custom.host.ranked Nie możesz hostować gier, gdy grasz w trybie rankingowym.
```
Comment (xhg4) [inline pl.js:815]: For the sake of consistency, I think "Nie możesz tworzyć rozgrywek..." or "Nie możesz utworzyć rozgrywki..." or maybe "Nie można tworzyć rozgrywek, grając w trybie rankingowym"

**Review by xhg4 — COMMENTED** [review]
Okay, I found 2 lines that could be changed but it's not really necessary
No idea how this many typos and even mistakes (like the one line were in the previous version only half of the phrase was translated) went unnoticed

**Comment by xhg4** [comment]
Btw, would you be willing to continue contributing to the Polish translation? If so, would you prefer to mostly do the translating or reviewing?

**Comment by ectryliz** [comment]
Can do both, depends how much free time I have.

**Comment by ectryliz** [comment]
@xhg4 There are some translations, that have word "rozgrywka". I think it would be better, if it got replaced with "gra". If you think the changes are wrong or seem extraordinary, I will revert them back.

File: pl.js (hunk 1 above)
Comment (ectryliz) [reply pl.js]: Missed this one

**Review by xhg4 — DISMISSED** [review]

**Comment by xhg4** [comment]
> @xhg4 There are some translations, that have word "rozgrywka". I think it would be better, if it got replaced with "gra". If you think the changes are wrong or seem extraordinary, I will revert them back.

Yeah, now that you mention it, I agree

If those are all the changes then it's ready to be merged @mty22

**Comment by ectryliz** [comment]
Everything here is probably done, it's ready to be merged.

**Review by xhg4 — APPROVED** [review]

**Comment by mty22** [comment]
@ectryliz can you please reach out to me with your IGN?

**Comment by mty22** [comment]
nevermind, got it.

**Pull request merged**